content/en/docs/setup/install/multicluster/single-network/verify/index.md

```diff
@@ -0,0 +1,318 @@
+        -l app=helloworld -n sample
+    {{< /text >}}
+
+Using the previously [set environment variables,](/docs/setup/install/multicluster/#env-var)
+deploy the HelloWorld service to `Cluster_2` with the following steps:
+
+1. Create the `sample` namespace in `Cluster_2` with the following command:
+
+    {{< text bash >}}
+    $ kubectl create --context=${CTX_2} namespace sample
+    {{< /text >}}
+
+1. Enable automatic sidecar injection for the `sample` namespace in
+   `Cluster_2` with the following command:
+
+    {{< text bash >}}
+    $ kubectl label --context=${CTX_2} namespace sample \
+        istio-injection=enabled
+    {{< /text >}}
+
+1. Deploy the `HelloWorld` service to `Cluster_2` with the following command:
+
+    {{< text bash >}}
+    $ kubectl create --context=${CTX_2} \
+        -f ${ISTIO}/samples/helloworld/helloworld.yaml \
+        -l app=helloworld -n sample
+    {{< /text >}}
+
+## Deploy different service versions to different clusters
+
+To verify cross-cluster load balancing, deploy `v1` of the `HelloWorld` service to
+`CLUSTER_1` and `v2` to `CLUSTER_2`.
+
+Using the previously [set environment variables,](/docs/setup/install/multicluster/#env-var)
+deploy the different versions of the service to the clusters with the following
+steps:
+
+1. Deploy `v1` of the `HelloWorld` service to `Cluster_1` with the following
+   command:
+
+    {{< text bash >}}
+    $ kubectl create --context=${CTX_1} \
+        -f ${ISTIO}/samples/helloworld/helloworld.yaml \
+        -l app=helloworld -l version=v1 -n sample
+    {{< /text >}}
+
+1. Deploy `v2` of the HelloWorld service to `Cluster_2` with the following
+   command:
+
+    {{< text bash >}}
+    $ kubectl create --context=${CTX_2} \
+        -f ${ISTIO}/samples/helloworld/helloworld.yaml \
+        -l app=helloworld -l version=v2 -n sample
+    {{< /text >}}
+
+1. Confirm the status `v1` of the HelloWorld service with the following
+   command:
+
+    {{< text bash >}}
+    $ kubectl get pod --context=${CTX_1} -n sample
+    NAME                            READY     STATUS    RESTARTS   AGE
+    helloworld-v1-86f77cd7bd-cpxhv  2/2       Running   0          40s
+    {{< /text >}}
+
+1. Wait until the status of `helloworld-v1` is `Running`.
+1. Confirm the status `v2` of the HelloWorld service with the following command:
+
+    {{< text bash >}}
+    $ kubectl get pod --context=${CTX_2} -n sample
+    NAME                            READY     STATUS    RESTARTS   AGE
+    helloworld-v2-758dd55874-6x4t8  2/2       Running   0          40s
+    {{< /text >}}
+
+1. After the status of `helloworld-v2` is `Running`, your cluster is ready
+   to deploy a load-generating service.
+
+## Deploy a load-generating service
+
+To generate the traffic needed to verify your deployment, deploy the `sleep`
+service to your clusters. The `sleep` service is a simple load-generating
+service. If you are using your own service to verify your deployment, ensure
```

Comment:
No it's not, it's a simple sleeping service.  you're (ab)using it as a pod you can use to curl things.  Why not use a curl service, or fortio or some other load generator perhaps?

Reply:
It's a valid point. Do we use curl or fortio anywhere else currently?

Reply:
I can rewrite the sentence to `We use the sleep service as a simple load-generating service.`

Reply:
The bigger point is that _we should not be using a pod called `sleep` to do something other than sleep._ If you want me to raise an issue for this as a general rule, please let me know.

Reply:
Maybe a bad name, but this [sleep service](https://github.com/istio/istio/blob/master/samples/sleep/README.md#simple-sleep-service) was designed for this purpose and is used pervasively in our task docs as a source service for demonstrating various Istio features. We use fortio, instead, for a source when we want to generate load.

Reply:
Just to be clear, the sleep service is not a "load-generating service". It's purpose to provide an "in-mesh pod" that users can exec into and use curl to make requests that are internal mesh (sidecar-to-sidecar) traffic.